content-services/latest/install/ansible.md

```diff
@@ -0,0 +1,489 @@
+---
+title: Install with Ansible
+---
+
+# Alfresco Ansible Installation
+
+This page describes how to install Alfresco Content Services (ACS) using an [Ansible](https://www.ansible.com) playbook. Ansible is an open-source software provisioning, configuration management and application installation tool enabling infrastructure as code. Alfresco provides an Ansible playbook capable of installing Alfresco Content Services (ACS) Enterprise 7.x or 6.2.N.
+
+Before continuing we need to introduce some [Ansible concepts](https://docs.ansible.com/ansible/latest/user_guide/basic_concepts.html); [control node](https://docs.ansible.com/ansible/latest/user_guide/basic_concepts.html#control-node), [connection type](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#connecting-to-hosts-behavioral-inventory-parameters) and the [inventory file](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#intro-inventory).
+
+The machine the playbook is run from is known as the **control node**. An **inventory** file is used to describe where the different Content Services should be installed on. 
+
+There are two types of installation: 
+* **Local** where all the components will be installed on the control node machine:
+
+![](../../images/deployment-type-local.png)
```

Comment:
Image is not visible and I think the image link is not specified correctly, should be something like:
    ![deployment-type-local]({% link content-services/images/deployment-type-local.png %})